bin/cypress

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+
+ENV["RAILS_ENV"] ||= "test"
+ENV["REMOTE_USER"] ||= "auser@stanford.edu"
+ENV["ROLES"] ||= "dlss:hydrus-app-administrators"
+require "pathname"
+require "cypress-rails"
+require Pathname.new(CypressRails::Config.new.dir).join("config/environment")
```

Comment:
Add config to run headlessly with Firefox rather than Electron, like we discussed.